src/style.css

```diff
@@ -0,0 +1,116 @@
+
+@font-face {
+  font-family: "Avenir" sans-serif;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+.input {
+  
+  font-family: "Avenir", sans-serif;
+  width: 100%;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  box-sizing: border-box;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+  display: block;
+}
+
+.input-big {
+
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 62px;
+}
+
+.input-small {
+
+  font-family: "Avenir", sans-serif;
+  box-sizing: border-box;
+  height: 42px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  padding-left: 33px;
+    
+}
+
+.big-search-form {
+
+  position: relative;
+  margin-top: 20px;
+}
+
+.image-search-big {
+
+  position: absolute;
+  top: 23px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+
+}
+
+.small-search-form {
+  
+  position: relative;
+  margin-top: 20px;
+}
+
+.image-search-small {
+  position: absolute;
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+
+}
+
+.input:default {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input:hover {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+
+}
+
+.input:focus {
+  box-sizing: border-box;
+  background-color: linear-gradient (180deg, #fff 0%, #f6f6f7 100%);
+  border: 0 1px solid #e1e7ed;
+  border-radius: 4px;
+}
+
+.input-big::placeholder {
+  font-weight: 300;
+  color: #3d4e61;
```

Comment:
when you use color more than 1 time create variable and use it instead of 3d4e61 and e1e7ed